docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -202,16 +242,17 @@ The following keys are recognized:
 
   Example: `sort-key fedora`
 
-* `linux` is the Linux kernel to spawn and as a path relative to file system
-  root. It is recommended that every distribution creates a machine id and
+* `linux` is the Linux kernel to spawn and is a path relative to the root of
+  the file system the boot entry snippet itself is located in. It is
```

Comment:
`linux` is the Linux kernel to spawn and is a path relative to the root of the file system containing the boot entry snippet itself